xml/System.Runtime.InteropServices/IDynamicInterfaceCastable.xml

```diff
@@ -60,11 +77,20 @@
         <Parameter Name="throwIfNotImplemented" Type="System.Boolean" />
       </Parameters>
       <Docs>
-        <param name="interfaceType">To be added.</param>
-        <param name="throwIfNotImplemented">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="interfaceType">The interface type.</param>
+        <param name="throwIfNotImplemented">Indicates if the function should throw an exception instead of returning <see langword="false" />.</param>
+        <summary>Called when an implementing class instance is cast to an interface type that is not contained in the class's metadata.</summary>
+        <returns>Whether or not this object can be cast to the given interface</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+This is called if casting this object to the given interface type would otherwise fail. Casting here means the IL isinst and castclass instructions in the case where they are given an interface type as the target type.
```

Comment:
```suggestion
This is called if casting this object to the given interface type would otherwise fail. Casting here means the IL `isinst` and `castclass` instructions in the case where they are given an interface type as the target type.
```